src/fusor/translator.py

```diff
@@ -199,7 +199,7 @@ def _are_fusion_partners_different(
         )
         return False
 
-    def _get_genomic_ac(self, chrom: str, build: Assembly) -> str:
+    def _get_genomic_ac(self, chrom: str, build: Assembly | str) -> str:
```

Comment:
I don't know if I've ever looked at this code before, so take this with a grain of salt. But my perhaps arbitrary stylistic preference is that stricter type signatures are better, and that this type coercion should happen closer to wherever the `str` value is coming from in the first place. Just my two cents.

Reply:
That's fair, I was optimizing for a shorter PR, but fixing the tests to pass the enum value instead would probably be better in the long run.

Reply:
Yup, totally reasonable. This wouldn't be the only place where we employ the `str | Enum` pattern. Feel free to ignore especially if the alternative is nontrivial